notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_inertia_compensation_single_slew_hp.ipynb

```diff
@@ -79,6 +79,7 @@
    "outputs": [],
```

Comment:
<p>Maybe need info. for BLOCK-146 as BLOCK-146 and -178 were queried below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>